www/app/blog/2024-11-19-distributed-ddl.mdx

```diff
@@ -0,0 +1,52 @@
+export const metadata = {
+  title: "Distributed DDL at Global Scale",
+  authors: ["gwenshap"],
+  image: "2024-11-19-distributed-ddl/cover.webp",
+  sizzle:
+    "TBD",
+  tags: ["database", "serverless", "postgres", "distributed systems"],
+};
+
+When we need to describe Nile in a single sentence, we say "PostgreSQL re-engineered for multi-tenant apps". By multi-tenant app we mean applications like Stripe, Figma, Twilio, Notion, Workday, Gusto... where a large number of customers is served off a single application stack. In this type of application, a key part of the architecture is deciding how to store the data for each customer. 
+
+Broadly speaking, there are two main approaches. The first is database per tenant (or sometimes schema per tenant):
+
+![Database per tenant architecture](/blog/2024-11-19-distributed-ddl/db_per_tenant.png)
+
+This architecture provides isolation and flexibility, but it uses more resources and more effort to operate. 
+
+The other approach is to place all tenants in the same schema, and add a tenant identifier to all the tables:
```

Comment:
should you explicitly name this approach as shared schema since that's the name used later on?